lib/go/thrift/simple_server.go

```diff
@@ -183,9 +186,6 @@ func (p *TSimpleServer) processRequests(client TTransport) error {
 			log.Printf("error processing request: %s", err)
 			return err
 		}
-		if err, ok := err.(TApplicationException); ok && err.TypeId() == UNKNOWN_METHOD {
-			continue
-		}
```

Comment:
It's removed by gofmt. I thought that gofmt knows better tan me, so I didn't revert that change. should I?